auth/auth/appsettings.json

```diff
@@ -6,7 +6,7 @@
     }
   },
   "AllowedHosts": "*",
-  "ConnectionStrings": {
-    "AuthPostgresConnection": "Server=localhost;Port=5011;Database=auth;Username=postgres;Password=password"
-  }
+    "ConnectionStrings": {
+        "AuthPostgresConnection": "Server=localhost;Port=5432;Database=auth;Username=postgres;Password=mysecretpassword"
+    }
```

Comment:
It is good to use different database ports in case of hosting full app on one public address. Additionally, default db password is "password"